app/furniture/journal/entry.rb

```diff
@@ -9,6 +9,7 @@ class Entry < ApplicationRecord
     scope :recent, -> { order("published_at DESC NULLS FIRST") }
 
     attribute :headline, :string
+    validates :headline, presence: true
```

Comment:
Not really part of your PR, but now that we have the `strip_attributes` gem, it might be worth throwing that on for `headline` and `body`, just so that we don't store trailing/preceding whitespace.

Reply:
Yah! I was thinking about that then I forgot 🤦